src/web/app/pages-session/session-submission-page/session-submission-page.component.spec.ts

```diff
@@ -1036,7 +1046,7 @@ describe('SessionSubmissionPageComponent', () => {
   });
 
   it('should delete participant comment', () => {
-    const testSubmissionForm: QuestionSubmissionFormModel = deepCopy(testMcqQuestionSubmissionForm);
+    const testSubmissionForm: QuestionSubmissionFormModel = deepCopy(testMsqQuestionSubmissionForm);
     const expectedId: any = testMcqQuestionSubmissionForm.recipientSubmissionForms[0]
```

Comment:
Let's change the name here. Make it consistent with testMsq as per previous line

Reply:
Fixed :)